github/event_types.go

```diff
@@ -7,6 +7,38 @@
 
 package github
 
+// CheckRunEvent is triggered when a check run is "created", "updated", or "re-requested".
+// The Webhook event name is "check_run".
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#checkrunevent
+type CheckRunEvent struct {
+	CheckRun *CheckRun `json:"check_run,omitempty"`
+	// The action performed. Can be "created", "updated" or "re-requested".
```

Comment:
requested_action is added.
https://developer.github.com/changes/2018-05-23-request-actions-on-checks/
https://developer.github.com/v3/activity/events/types/#checkrunevent